etc/certs/generate.sh

```diff
@@ -46,6 +46,10 @@ openssl x509 -req -days 9999 -CAcreateserial -extfile <(printf "subjectAltName=D
 openssl req -new -newkey rsa:2048 -nodes -keyout hostcert_minio.key.pem -subj "/CN=minio" > hostcert_minio.csr
 openssl x509 -req -days 9999 -CAcreateserial -extfile <(printf "subjectAltName=DNS:minio,DNS:localhost") -in hostcert_minio.csr -CA rucio_ca.pem -CAkey rucio_ca.key.pem -out hostcert_minio.pem -passin env:PASSPHRASE
 
+# SSH server
+mkdir ../../etc/ssh
```

Comment:
this relative notation is dangerous :-)

in general this generate.sh is not to be used "automatically"... but to copy/paste the necessary stuff in your local console.  (maybe worth adding a comment about this at the top)

i think it's ok to just create the keys in the current directory

Reply:
@mlassnig okay, ill change it everywhere under ssh directory within certs itself

Reply:
@mlassnig done!